layers/+window-management/spacemacs-layouts/funcs.el

```diff
@@ -117,6 +122,25 @@ perspectives does."
                       (projectile-switch-project-by-name project)))))))
    :buffer "*Helm Projectile Layouts*"))
 
```

Comment:
You've forgotten to `(push project layouts-projectile)` in `spacemacs/helm-persp-switch-project`. :wink: